src/utils/verifyFrame.ts

```diff
@@ -21,6 +21,10 @@ export async function verifyFrame({
   url,
 }: VerifyFrameParameters): Promise<VerifyFrameReturnType> {
   const body = hexToBytes(`0x${trustedData.messageBytes}`)
+  if (hub.verifyFrame) {
+    return await hub.verifyFrame({ hub, trustedData, body, frameUrl, url })
+  }
+
```

Comment:
I think it could be better to do something like 

```tsx
const response = 
  await hub.verifyFrame({ hub, trustedData, body, frameUrl, url }) ||
  await fetch(`${hub.apiUrl}/v1/validateMessage`, {
    ...hub.fetchOptions,
    method: 'POST',
    headers: {
      'Content-Type': 'application/octet-stream',
      ...hub.fetchOptions?.headers,
    },
    body,
  }).then((res) => res.json())
```

This means we can remove the implementation detail on [L37-45](https://github.com/wevm/frog/pull/292/files#diff-84ea46dbca6bd046b837ede5a9b7cd8add419d453d1113e543034ca636fe2405R37-R45) and `return` after the `fetch` call.